scene/gui/label.cpp

```diff
@@ -627,6 +630,17 @@ int Label::get_total_character_count() const {
 	return total_char_cache;
 }
 
+int Label::get_page_character_count() const {
+
+	// default to total_char_cache
+	// should be correct on the next draw
+	if (word_cache_dirty) {
+		const_cast<Label *>(this)->regenerate_word_cache();
```

Comment:
Why is the casting necessary?

Reply:
I just follow `get_total_character_count` function.

Reply:
Casting is necessary as `regenerate_word_cache` is not const, while this function is.

Reply:
Thanks for the clarification :)